src/main/java/de/mediathekview/mserver/crawler/orf/tasks/OrfHelper.java

```diff
@@ -0,0 +1,55 @@
+package de.mediathekview.mserver.crawler.orf.tasks;
+
+import de.mediathekview.mserver.base.Consts;
+import de.mediathekview.mserver.crawler.orf.OrfConstants;
+import java.util.ArrayList;
+import java.util.List;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+/**
+ * Helper methods for ORF tasks
+ */
+public class OrfHelper {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)